pkg/yang/entry.go

```diff
@@ -110,6 +110,13 @@ type RPCEntry struct {
 	Output *Entry
 }
 
+// A ListAttr is associated with an Entry that represents a List node
+type ListAttr struct {
+	MinElements *Value // leaf-list or list MUST have at least min-elements
+	MaxElements *Value // leaf-list or list has at most max-elements
+	OrderedBy   *Value // order of entries determined by "system" or "user"
+}
+
```

Comment:
You mean ListAttr or the below functions? ListAttr is just moved to get it out of the way to Modules(). The functions below are used in ocstructs which is slated for OS release soon - that's where the change will be made to use the functions here instead.